internal/cmd/postgresflex/instance/backups/describe/describe.go

```diff
@@ -0,0 +1,140 @@
+package describe
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/depp/bytesize"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/tables"
+	"github.com/stackitcloud/stackit-sdk-go/services/postgresflex"
+)
+
+const (
+	backupIdArg = "BACKUP_ID"
+
+	instanceIdFlag = "instance-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	InstanceId string
+	BackupId   string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("describe %s", backupIdArg),
+		Short: "Shows details of a backup for a specific PostgreSQL Flex instance",
+		Long:  "Shows details of a backup for a specific PostgreSQL Flex instance.",
+		Example: examples.Build(
+			examples.NewExample(
+				`Get details of a backup with ID "xxx" for a PostgreSQL Flex instance with ID "yyy"`,
+				"$ stackit postgresflex instance backups describe xxx --instance-id yyy"),
+			examples.NewExample(
+				`Get details of a backup with ID "xxx" for a PostgreSQL Flex instance with ID "yyy" in a table format`,
+				"$ stackit postgresflex instance backups describe xxx --instance-id yyy --output-format pretty"),
+		),
+		Args: args.SingleArg(backupIdArg, nil),
```

Comment:
```suggestion
		Args: args.SingleArg(backupIdArg, utils.ValidateUUID)),
```

We should validate that the provided backup id is a UUID

Reply:
Backup ID does not have to be a UUID, that is why I did not add validation. 
I also verified it now with the Postgres team.